pkg/cmd/api-request-count/api_request_count.go

```diff
@@ -0,0 +1,153 @@
+package api_request_count
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	apiv1 "github.com/openshift/api/apiserver/v1"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog/v2"
+)
+
+var (
+	example = `
+    # counts resources used by users and generates a static HTML page.
+    #
+    # "bob": {
+    #  "configmaps":1,
+    #  "secrets":2,
+    # }
+    %[1]s apicount -f path/to/apirequestcounts --tmpl path/to/html/template
+
+	# counts users of a resource and generates a static HTML page.
+    #
+    # "secrets": {
+    #   "bob":1,
+    #   "alice":2,
+    # }
+	%[1]s apicount --by resource -f path/to/apirequestcounts --tmpl path/to/html/template
+`
+)
+
+func NewCmdAPIRequestCount(parentName string, streams genericclioptions.IOStreams) *cobra.Command {
+	o := apiRequestCountOptions{}
+
+	cmd := &cobra.Command{
+		Use:          "apicount [flags]",
+		Short:        "Creates a static HTML page for api requests",
+		Example:      fmt.Sprintf(example, parentName),
+		SilenceUsage: true,
+		RunE: func(c *cobra.Command, args []string) error {
+			if err := o.Complete(c, args); err != nil {
+				return err
+			}
+			if err := o.Validate(); err != nil {
+				return err
+			}
+			if err := o.Run(); err != nil {
+				return err
+			}
+
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVarP(&o.inputDirectory, "datadir", "f", o.inputDirectory, "A directory which contains api requests data")
+	cmd.Flags().StringVarP(&o.outputDirectory, "outdir", "o", o.outputDirectory, "The path of the output directory")
```

Comment:
output file seems more appriopriate.  We're going for a single html file to make it loadable in CI output.